### PR TITLE
[11.0][MIG] sql_request_abstract

### PR DESCRIPTION
**Pull request by HviorForgeFlow**

### CODE REVIEW

File: sql_request_abstract/security/ir.model.access.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_sql_request_mixin_all,access_sql_request_mixin_all,model_sql_request_mixin,,0,0,0,0
-access_sql_request_mixin_user,access_sql_request_mixin_user,model_sql_request_mixin,sql_request_abstract.group_sql_request_user,1,0,0,0
-access_sql_request_mixin_manager,access_sql_request_mixin_manager,model_sql_request_mixin,sql_request_abstract.group_sql_request_manager,1,1,1,1
+access_sql_request_mixin_all,all_sql_request_mixin,model_sql_request_mixin,,0,0,0,0
+access_sql_request_mixin_user,user_sql_request_mixin,model_sql_request_mixin,sql_request_abstract.group_sql_request_user,1,0,0,0
+access_sql_request_mixin_manager,manager_sql_request_mixin,model_sql_request_mixin,sql_request_abstract.group_sql_request_manager,1,1,1,1
```
Comment (legalsylvain) [inline sql_request_abstract/security/ir.model.access.csv]: not sure to see the interest of such changes ?

**Review by legalsylvain — COMMENTED** [review]
Hi @hveficent. thanks for porting this module.
~Could you cherry pick (and review) this PR, related to the module you're porting ?~
~https://github.com/OCA/server-tools/pull/1312~
kind regards.

File: sql_request_abstract/security/ir.model.access.csv (hunk 1 above)
Comment (HviorForgeFlow) [reply sql_request_abstract/security/ir.model.access.csv]: No one, just had a few errors with that file, but now I don't.

**Review by legalsylvain — COMMENTED** [review]
:+1: 

(Note : Travis is failing for unrelated reason. (and broken on 11.0 branch))

**Review by legalsylvain — APPROVED** [review]

**Comment by nicomacr** [comment]
Hi @hveficent thaks for the PR, i've tested it with the `bi_sql_editor` module and has appeared an error when execute an sql query wrong.
`_("The SQL query is not valid:\n\n %s") % e.message)
AttributeError: 'ProgrammingError' object has no attribute 'message'`
Kind regards

**Comment by HviorForgeFlow** [comment]
@nicomacr I think it is solved now, Thanks :smile:

**Review by nicomacr — APPROVED** [review]

**Comment by nicomacr** [comment]
+1

**Review by MiquelRForgeFlow — APPROVED** [review]

File: sql_request_abstract/README.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================================
+Abstract Model to manage SQL Requests
+=====================================
+
+This module provide an abstract model to manage SQL Select request on database.
```
Comment (rvalyi) [inline sql_request_abstract/README.rst]: s /provide/provides
s /request/requests/

Comment (HviorForgeFlow) [reply sql_request_abstract/README.rst]: Thank for reviewing, comments attended.

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Reviewed through https://github.com/OCA/reporting-engine/pull/230

**Pull request merged**